pkg/e2e/operators/managedvelero.go

```diff
@@ -29,4 +36,37 @@ var _ = ginkgo.Describe("[Suite: operators] [OSD] Managed Velero Operator", func
 	checkRoleBindings(h,
 		operatorNamespace,
 		[]string{"managed-velero-operator"})
+	checkVeleroBackups(h)
 })
+
+func checkVeleroBackups(h *helper.H) {
+	ginkgo.Context("velero", func() {
+		ginkgo.It("backups should be complete", func() {
+			err := wait.PollImmediate(5*time.Second, 5*time.Minute, func() (bool, error) {
+				us, err := h.Dynamic().Resource(schema.GroupVersionResource{
+					Group:    "velero.io",
+					Version:  "v1",
+					Resource: "backups"}).Namespace(operatorNamespace).List(context.TODO(), metav1.ListOptions{})
+				if err != nil {
+					return false, fmt.Errorf("Error getting backups: %s", err.Error())
+				}
+				for _, item := range us.Items {
+					var backup velerov1.Backup
+					err = runtime.DefaultUnstructuredConverter.
+						FromUnstructured(item.UnstructuredContent(), &backup)
+					if err != nil {
+						return false, fmt.Errorf("Error casting object: %s", err.Error())
+					}
+					if backup.Status.Phase == velerov1.BackupPhaseFailed {
+						return false, fmt.Errorf("Backup failed for %s", backup.Name)
+					}
+					if backup.Status.Phase != velerov1.BackupPhaseCompleted {
+						return false, nil
+					}
```

Comment:
This loops through the backups, and if any one of them is in this state it immediately returns out a false and fails the test. Do we want to check again if something isn't complete? Either way, we should also return out which bucket is not completed if we are going to fail the test completely.